checks.d/powerdns-recursor.py

```diff
@@ -0,0 +1,40 @@
+# Datadog
+from checks import AgentCheck
+
+# Other
+import json
+import urllib2
+from collections import namedtuple
+
+class PdnsChecks(AgentCheck):
+
+
+    def _get_pdns_stats(self, instance):
+        config = self._get_config(instance)
+        request = urllib2.Request(config.pdns_url + ':' + str(config.port) + "/servers/localhost/statistics", headers={"X-API-Key" : config.api_key})
+        content = urllib2.urlopen(request).read()
+        result = json.loads(content)
+        return result
```

Comment:
Do you have an example of a result ? How many stats would be returned most of the time ?